src/internal/operations/restore.go

```diff
@@ -167,6 +168,11 @@ func (op *RestoreOperation) Run(ctx context.Context) (restoreDetails *details.De
 	if err != nil {
 		// No return here!  We continue down to persistResults, even in case of failure.
 		logger.CtxErr(ctx, err).Error("running restore")
+
+		if errors.Is(err, kopia.ErrNoRestorePath) {
+			op.Errors.Fail(clues.New("running restore: empty backup or unknown path provided"))
```

Comment:
Try to avoid prefixing errors with categorizations like `running restore:`.  It's the job of upstream handlers to wrap error messages with that prefix.  Using it in lower level messages can result in messages that read like this, `running restore: foo: bar: running restore: baz: qux`.  And that just ends up being confusing.

Reply:
done